imports/api/detectors/methods.js

```diff
@@ -0,0 +1,72 @@
+import { Meteor } from 'meteor/meteor';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+import { _ } from 'meteor/underscore';
+
+
+export const matchAffordancesWithDetector = new ValidatedMethod({
```

Comment:
kapil and i have been switching to just regular javascript functions, no need to use the validator schema any more especially for a function that will only live on the server and be called by the server

Reply:
I need help writing a test case so I can test this method on the server.